assets/helpers/contributions.js

```diff
@@ -442,6 +444,18 @@ function getFrequency(contributionType: Contrib): string {
 
 }
 
+function getPaymentType(contributionType: Contrib, paymentType: PaymentType): string {
```

Comment:
This method name is a bit confusing as it takes a payment type as an argument. It could be e.g. `getPaymentTypeQualifier`

Reply:
Agree. It's used for text on a button? Would `getPaymentDescription` also work?